# Generated by Django 3.2 on 2022-04-17 14:23

import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('walker_profile', '0010_addressdetails_created_at'),
    ]

    operations = [
        migrations.AlterField(
            model_name='servicedetails',
            name='b_price_day',
            field=models.PositiveIntegerField(default=0, null=True, validators=[django.core.validators.MaxValueValidator(10)]),
        ),
        migrations.AlterField(
            model_name='servicedetails',
            name='b_price_hour',
            field=models.PositiveIntegerField(default=0, null=True, validators=[django.core.validators.MaxValueValidator(10)]),
        ),
        migrations.AlterField(
            model_name='servicedetails',
            name='m_price_day',
            field=models.PositiveIntegerField(default=0, null=True, validators=[django.core.validators.MaxValueValidator(10)]),
        ),
        migrations.AlterField(
            model_name='servicedetails',
            name='m_price_hour',
            field=models.PositiveIntegerField(default=0, null=True, validators=[django.core.validators.MaxValueValidator(10)]),
        ),
        migrations.AlterField(
            model_name='servicedetails',
            name='s_price_day',
            field=models.PositiveIntegerField(default=0, null=True, validators=[django.core.validators.MaxValueValidator(10)]),
        ),
        migrations.AlterField(
            model_name='servicedetails',
            name='s_price_hour',
            field=models.PositiveIntegerField(default=0, null=True, validators=[django.core.validators.MaxValueValidator(10)]),
        ),
    ]
